test/testsuites/defaultcatsrctests.go

```diff
@@ -143,69 +151,68 @@ func testDefaultCatsrcWhileDisabled(t *testing.T) {
 			APIVersion: "operators.coreos.com/v1alpha1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "redhat-operators",
+			Name:      catSrcName,
 			Namespace: namespace,
 		},
 		Spec: olm.CatalogSourceSpec{
-			SourceType:  "grpc",
+			SourceType: "grpc",
+			// TODO(tflannag): Do we care that this image will cause an ImagePullBackOff?
```

Comment:
Just mirroring the TODO comment here - in the context of this test I don't think we care as we're just ensuring the reconciliation behavior but using this image will result in the CatalogSource Pod being deployed with an image we can't authenticate to and the result is the Pod lands in the `ImagePullBackOff` state and we don't verify that Pod is healthy after updating.